.env

```diff
@@ -0,0 +1,14 @@
+alias ftest='python -m pytest'
+
+alias gpush='git push origin'
+alias gpull='git pull'
+alias gfetch='git fetch'
+alias gstash='git stash'
+alias grebase='git rebase'
+alias grebase_master='git rebase -i master'
+alias grebase_develop='git rebase -i develop'
+alias grebase_continue='git rebase --continue'
+alias checkout='git checkout'
+alias branch='git branch'
+alias cdev='git checkout develop'
+alias cmaster='git checkout master'
```

Comment:
@sengineer0  do you want to remove this file from the commit?

Reply:
Srr, my mistake. I thought I added this `.env` file to `.gitignore`. Will remove it soon.

Reply:
`@sengineer0 are we plan to get away from using utils.es module?`

I think it is a complexity task, if we want to remove `utils.es` module, it should be break into smaller tasks, like: completely replace the utils.es.ESIndexer` with the `dataindex.indexer.Indexer`, remove the usage of `utils.es` anywhere in the codebase, and replace with other functions/utils, ...

But currently, I dont have fully understand the `utils.es`, and `dataindex.indexer`
They are too complex.

Reply:
@sengineer0 how about you create your own `autoupdate.indexer` model after `dataindex.indexer`, create only those methods you need for `autoupdate`? That way, we can remove the use of `utils.es` one by one, later we can re-factor these new `indexer` modules into a new common `ESIndexer` class.

Reply:
Good idea. I will give it a try.